.buildkite/pipeline-resource-definitions/kibana-coverage-daily.yml

```diff
@@ -0,0 +1,44 @@
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: bk-kibana-code-coverage-main
+  description: 'Collects code coverage for unit and e2e tests, publishes results on Kibana stats cluster'
+  links:
+    - url: 'https://buildkite.com/elastic/kibana-code-coverage-main'
+      title: Pipeline link
+spec:
+  type: buildkite-pipeline
+  owner: 'group:appex-qa'
+  system: buildkite
+  implementation:
+    apiVersion: buildkite.elastic.dev/v1
+    kind: Pipeline
+    metadata:
+      name: kibana / code-coverage / main
+      description: 'Collects code coverage for unit and e2e tests, publishes results on Kibana stats cluster'
+    spec:
+      env:
+        SLACK_NOTIFICATIONS_CHANNEL: '#appex-qa-bots'
+        GITHUB_COMMIT_STATUS_CONTEXT: kibana-code-coverage-main
```

Comment:
Is this going to do anything?

Reply:
I copied it from the pipeline's current setup (https://buildkite.com/elastic/kibana-code-coverage-main/settings/steps) - and it seems to be used by scripts in the post-build routine: https://github.com/elastic/kibana/blob/1de8c74a68f8a7f3065ff1a7423dc27e696b4b72/.buildkite/scripts/lifecycle/post_build.sh#L9
I don't currently have the full context to make the judgement, if it's useful. Do you think it's irrelevant here?